web-ui/src/pages/analytics/programs.tsx

```diff
@@ -2,15 +2,14 @@ import Grid from '@mui/material/Grid'
 import Typography from '@mui/material/Typography'
 import PageHeader from 'src/layouts/components/page-header'
 import TableSqlPrograms from 'src/analytics/TableSqlPrograms'
+import { usePageHeader } from 'src/compositions/ui/pageTitle'
 
 const SqlPrograms = () => {
+  usePageHeader(s => s.setHeader)(
+    <PageHeader title='SQL Programs' subtitle='View status and edit already defined SQL programs.' />
```

Comment:
It would be nice if we can make this so we just set usePageHeader(title='bla, subtitle='bla') as strings, and the internals of usePageHeader use the default <PageHeader ...> JSX.Element to display

but then also be able to override the default PageHeader if we need something custom (e.g., for the breadcrumb/data-insertion browser view)